Simplenote/src/main/java/com/automattic/simplenote/NoteEditorActivity.java

```diff
@@ -76,7 +76,6 @@
 
```

Comment:
We can remove the `import com.automattic.simplenote.utils.ThemeUtils;` statement since it is no longer used.